### DISCUSSION TITLE
General question - Generating different results on the same dataset

### DISCUSSION THREAD

**Discussion by s-mostafa-a** [stella-cv/stella_vslam#324] [Q&A] [answered]
Hi everyone, 
Thank you for your great effort in developing this repository. I was tracking the development since it was the community version of openvslam.
I have a general question about this specific SLAM since I am quite new to this business. What are the possible reasons that it generates different results for the same dataset in each run? Is it because of the optimization steps which it uses random values each time?

Thank you in advance.

**Comment by ymd-stella** [comment, marked as answer]
I am not interested in strictly matching results. However, contributions are welcome.
